templates/docs/whats-new.html

```diff
@@ -0,0 +1,547 @@
+{% extends "_layouts/docs.html" %}
+
+{% block content %}
+
+<h1>  What's new in Vanilla {{versionMinor}} </h1>
+
+<hr>
+
+When we add, make significant updates, or deprecate a component we update their status so that it’s clear what’s available to use. Check back here anytime to see current status information.
+
+<table aria-label="What's new in Vanilla {{versionMinor}}">
+  <thead>
+    <tr>
+      <th style="width: 20%">Component</th>
+      <th style="width: 15%">Status</th>
+      <th style="width: 10%">Version</th>
+      <th style="width: 55%">Notes</th>
+    </tr>
+  </thead>
+  <tbody>
+    <!-- 3.12.0 -->
+    {% for feature in releaseNotes[0].features %}
+    <tr>
+      <th><a href="{{ feature.url }}">{{ feature.component|safe }}</a></th>
+      <td>
+        {% if feature.status=="New" %}
+        <span class="p-status-label--positive">
+        {% elif feature.status=="Updated" %}
+        <span class="p-status-label--information">
+        {% elif feature.status=="Deprecated" %}
+        <span class="p-status-label--negative">
+        {% endif %}
```

Comment:
Here we're checking fewer status than in the side nav. Is that correct?

Reply:
Yes. Not checking Removed here. That is in a separate table at the bottom

Reply:
The separate table at the bottom is for things removed in 3.0.0, if anything is removed later it technically should appear normally on the list.

But, because of backwards compatibility we are not going to remove anything until 4.0.0 is released. So I guess it doesn't matter that much now.

Reply:
I added all the removed stuff. But ideally, from 4.0.0, we can start documenting in which exact version something was removed

Reply:
We never remove anything in versions other than major.

So in 4.0.0 we will remove whatever is deprecated so far. Later new things will get deprecated in 4.X versions, to be removed later in 5.0.0.